### ISSUE TITLE
Archiving this repo

### ISSUE THREAD

**Issue by mattip** [MacPython/h5py-wheels#14]
I got a notice that there may have been a security leak of credentials connected to this repo. I see that [h5py](https://github.com/h5py/h5py) is using cibuildwheel. Is it still in use? Can we archive it?

**Comment by matthew-brett** [comment]
Just to say - I got that notification too - and I immediately turned on OAuth app restrictions for MacPython - as the email recommended.

**Comment by mattip** [comment]
ping @takluyver: any thoughts?

**Comment by takluyver** [comment]
We currently build wheels for Mac x64 and ARM on Azure from the main h5py repo, and that's what we upload to PyPI. I'm always a bit split on this - all the stuff we do in CI, including making wheels, makes it quite slow. But it means we're testing the wheel builds frequently, so we're less likely to get a nasty surprise at release time (or worse, afterwards). So far, I'm happy enough with this trade-off.

TL;DR: yes, I'm OK for you to archive this repo.

**Comment by mattip** [comment]
Thanks. I will archive it.
